data-es6.js

```diff
@@ -272,7 +272,7 @@ exports.browsers = {
   firefox37: {
     full: 'Firefox',
     short: 'FF 37',
-    obsolete: very,
```

Comment:
It was not on purpose, I removed these lines during my testing and copied back from FF36 when I finished.  But the comma is not necessary on the last item, and all other versions above this don't have it.